docs/nirfsg/rep_caps.rst

```diff
@@ -101,16 +101,42 @@ waveform
         passes a string of :python:`'waveform::0, waveform::1, waveform::2'` to the set attribute function.
 
 
-deembedding_port
-----------------
+port
+----
 
-    .. py:attribute:: nirfsg.Session.deembedding_port[]
+    .. py:attribute:: nirfsg.Session.port[]
 
         .. code:: python
 
-            session.deembedding_port['0-2'].channel_enabled = True
+            session.port['0-2'].channel_enabled = True
 
         passes a string of :python:`'0, 1, 2'` to the set attribute function.
 
 
+lo
```

Comment:
>lo

Is lo the right name for this rep cap?
We usually (always?) make our rep cap names plural and I'm not sure how I feel about abbreviating them, even if the abbreviation is very common in the API.

los?
local_oscillators?

Thoughts, @marcoskirsch?